coilsnake/model/common/ips.py

```diff
@@ -70,7 +70,7 @@ def is_applied(self, rom):
                         return False
             elif instruction == 'RECORD':
                 offset, size, data = args
-                if rom[offset:offset + size] != data:
+                if rom[offset:offset + size].to_list() != data:
```

Comment:
I suggest adding another method to Block here that explicitly returns a list instead of a Block